### ISSUE TITLE
Explore options for customizing font faces for AMP Stories

### ISSUE THREAD

**Issue by alcurrie** [ampproject/amp-wp#1643]
**As a AMP story author, I want to be able to customize the fonts used for my AMP story**

- [ ] _AC 1:_ Explore option to customize font faces (ie. maybe Google fonts)
- Related to #1565 - Larger default styles and #1567 (This issue was previously part of #1567  but has been separated out here the discussion/follow up does not block the other work on that issue.)

NB: Per discussion in Slack this AC may be blocked by Gutenberg see note in Slack re. ongoing discussion referenced by @jwold here:  WordPress/gutenberg#11648 and
google/fonts#1495

**Comment by alcurrie** [comment]
@miina  - as discussed, I've broken this out into a separate issue for review discussion.

**Comment by alcurrie** [comment]
Thanks @mehigh  - I think that sounds like a good approach to me from a user perspective. Do you have a sense of how time consuming and/or complex this would be to implement?   

@miina  - do you have any concerns about this re. implementing/integrating with existing deliverables this week?

**Comment by mehigh** [comment]
@alcurrie - probably around a day. This doesn't exactly align with the deadlines. 
Maybe if Miina thinks it's OK, we can take it into consideration as a - if we have the time.

**Comment by miina** [comment]
@mehigh Thank you for looking into this!

What would be relatively simple to do would be to add a new Select control for text blocks from where the user could choose the font-family. And then we could add this as an inline style for the relevant block. If this sounds like a suitable solution for text blocks then we might have time to add it, do you think that would be OK, to add the font family as inline style?

Would be great to have a list of suitable font families to add to the list.

We could leave this as the last ticket to resolve as we get all the critical issues done. Thoughts, @alcurrie?

**Comment by alcurrie** [comment]
@miina / @mike That makes sense to me. Let's add this to the discussion at backlog grooming today.

**Comment by jacobschweitzer** [comment]
I've done the work on this locally but am getting permissions errors when trying to push the code now. 
@miina @westonruter I've signed the CLA just a little while ago, maybe I have to wait for that to go through?

**Comment by jacobschweitzer** [comment]
@miina  Could you take a look when you get a chance? Thanks!
https://github.com/ampproject/amp-wp/pull/1713

**Comment by miina** [comment]
Closed via #1713

**Comment by alcurrie** [comment]
I've confirmed the update to allow the user to update font sizes. It looks great @jacobschweitzer  and @miina https://cl.ly/15ebb47eb736
